app/views/two_factor_authentication/shared/max_login_attempts_reached.html.erb

```diff
@@ -0,0 +1,18 @@
+<% lockout_time_in_words = decorated_user.lockout_time_remaining_in_words %>
+<% title t('titles.account_locked') %>
+
+<h1 class="h3 my0">
+  <%= t('titles.account_locked') %>
+</h1>
+<p>
+  <%= t('devise.two_factor_authentication.max_login_attempts_reached') %>
+</p>
+<p>
+  <%= t('devise.two_factor_authentication.please_try_again_html',
+  time_remaining: content_tag(:span, lockout_time_in_words, id: 'countdown')) %>
```

Comment:
Yes, that is the magic ✨  -- what I meant to convey is that, as someone who *does* know this magic, if I saw a translation without HTML but with `_html`, I might want to refactor to remove `_html` since it would not be necessary.